plugins/brjs-plugins/src/main/java/org/bladerunnerjs/plugin/require/AliasDataRequirePlugin.java

```diff
@@ -0,0 +1,40 @@
+package org.bladerunnerjs.plugin.require;
+
+import org.bladerunnerjs.api.Asset;
+import org.bladerunnerjs.api.BRJS;
+import org.bladerunnerjs.api.BundlableNode;
+import org.bladerunnerjs.api.model.exception.RequirePathException;
+import org.bladerunnerjs.api.plugin.RequirePlugin;
+import org.bladerunnerjs.api.plugin.base.AbstractRequirePlugin;
+import org.bladerunnerjs.plugin.plugins.require.AliasDataSourceModule;
+
+
+public class AliasDataRequirePlugin extends AbstractRequirePlugin implements RequirePlugin
+{
+	
+	private RequirePlugin defaultRequirePlugin;
+
+	@Override
+	public void setBRJS(BRJS brjs)
+	{
+		this.defaultRequirePlugin = brjs.plugins().requirePlugin("default");
+	}
+
+	@Override
+	public String getPluginName()
+	{
+		return "alias";
+	}
+
+	@Override
+	public Asset getAsset(BundlableNode bundlableNode, String requirePathSuffix) throws RequirePathException
```

Comment:
@dchambers We were previously getting `AmbiquousRequirePath` exceptions because `alias!$data`) is registered for each `BundlableNode` and the scope for an aspect bundle includes 2 bundlable nodes (the Aspect and the test pack). The alias data plugin ensures that when the alias data is required only the alias data for the current bundlable node is returned.